packages/@aws-cdk/cloudformation-include/test/valid-templates.test.ts

```diff
@@ -75,18 +74,18 @@ describe('CDK Include', () => {
       resources: [cfnBucket.attrArn],
     }));
 
-    expect(stack).toHaveResourceLike('AWS::IAM::Policy', {
+    Template.fromStack(stack).hasResourceProperties('AWS::IAM::Policy', {
       "PolicyDocument": {
         "Statement": [
-          {
+          Match.objectLike({
```

Comment:
I _think_ with Rico's changes (https://github.com/aws/aws-cdk/pull/18525), this addition should no longer be needed. Can you verify?